logging/log.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 Fraunhofer AISEC
```

Comment:
```suggestion
// Copyright 2016-2022 Fraunhofer AISEC
```

Reply:
New files should also have 2016-2020? Not the year they were created?

Reply:
yes (2016-2022), I will do a PR to update all headers accordingly.